iis-wf/iis-wf-ingest-pmc/src/test/java/eu/dnetlib/iis/wf/ingest/pmc/metadata/AffiliationMetadataExtractorMain.java

```diff
@@ -0,0 +1,73 @@
+package eu.dnetlib.iis.wf.ingest.pmc.metadata;
+
+import java.io.ByteArrayOutputStream;
+
+import org.apache.avro.io.DatumWriter;
+import org.apache.avro.io.Encoder;
+import org.apache.avro.io.EncoderFactory;
+import org.apache.avro.specific.SpecificDatumWriter;
+import org.apache.commons.lang.StringUtils;
+import org.jdom.Element;
+import org.xml.sax.SAXException;
+
+import eu.dnetlib.iis.common.importer.CermineAffiliation;
+import eu.dnetlib.iis.common.importer.CermineAffiliationBuilder;
+import eu.dnetlib.iis.ingest.pmc.metadata.schemas.Affiliation;
+import pl.edu.icm.cermine.exception.AnalysisException;
+import pl.edu.icm.cermine.exception.TransformationException;
+import pl.edu.icm.cermine.metadata.affiliation.CRFAffiliationParser;
+
+/**
+ * 
+ * Simple command line tool accepting affiliation string, to be provided as a single cmdline argument 
+ * (so needs to be specified in quotes), and producing extracted affiliations metadata.
+ * 
+ * @author mhorst
+ */
+public class AffiliationMetadataExtractorMain {
+
+    private static final String encoding = "utf-8";
+    
+    private static final CermineAffiliationBuilder cermineAffiliationBuilder = new CermineAffiliationBuilder();
+    private static final CermineToIngestAffConverter cermineToIngestAffConverter = new CermineToIngestAffConverter();
+    
+    public static void main(String[] args) throws Exception {
+        if (args.length >= 1) {
```

Comment:
Since it only uses one argument it should check that there is exactly one. It will also help to catch forgotten quoting.